pkgpanda/docs/filesystem_layout.md

```diff
@@ -1,54 +1,56 @@
 # File System Layout
-
-Host overall filesystem layout.
+pkgpanda packages expect a file system organized as described here so that symlinks
+may be generated with the bootstrap tarball before actually being placed on the host system.
 
 ```bash
-/etc/mesosphere/roles/{master,slave}
+/etc/mesosphere/roles/{master,slave,aws,aws_master}
 /etc/mesosphere/setup-flags/
-    active.json
     repository-url
-/etc/systemd/dcos.target.wants/
+/etc/systemd/system/dcos.target.wants/
     mesos-master.service
 /opt/mesosphere/
     bin/
     lib/
+    include/
     etc/
-    environment
     active/
-        mesos -> /opt/mesosphere/packages/mesos--0.22.0
-        java -> ...
-        zookeeper -> ...
+        mesos -> /opt/mesosphere/packages/mesos--deadbeefpackage
```

Comment:
We do not do human readable versions. Everything is a giant ugly hash. All throughout this documentation is references that describe these names as including human-readable versions but I have never seen anything but a hash